src/lib.rs

```diff
@@ -354,200 +355,211 @@ impl FileDialog {
     let mut command: Option<Command> = None;
 
     // Top directory field with buttons.
-    ui.horizontal(|ui| {
-      ui.add_enabled_ui(self.path.parent().is_some(), |ui| {
-        let response = ui.button("⬆").on_hover_text_at_pointer("Parent Folder");
-        if response.clicked() {
-          command = Some(Command::UpDirectory);
-        }
-      });
-      ui.with_layout(Layout::right_to_left(egui::Align::Center), |ui| {
-        let response = ui.button("⟲").on_hover_text_at_pointer("Refresh");
-        if response.clicked() {
-          command = Some(Command::Refresh);
-        }
+    egui::TopBottomPanel::top("top_panel").show_inside(ui, |ui| {
```

Comment:
Making this `egui::TopBottomPanel::top("egui_file_top_panel")` should be sufficiently unique.

Reply:
Please use the same prefix for the other panels too.